### PR TITLE
Add release 4.6.0 to releases menu

### PR DESCRIPTION
**Pull request by sijie**
Descriptions of the changes in this PR:

The documentation for release `4.6.0` was copied by #696. But this release is not updated in the releases menu. 

This change is generated by running `site/scripts/release.sh` for preparing 4.6.0 release.

### CODE REVIEW

File: site/docs/4.6.0/admin/http.md [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+    
+        | Code   | Description |
+        |:-------|:------------|
+        |200 | Successful operation |
+        |403 | Permission denied |
+        |404 | Not found |
+
+## Ledger
+
+### Endpoint: /api/v1/ledger/delete/?ledger_id=<ledger_id>
```
Comment (jiazhai) [inline site/docs/4.6.0/admin/http.md]: seems <> need some escape. sorry for this.

**Comment by jiazhai** [comment]
Thanks a lot for the help on this. Running "make build" and "make serve" for the site, It looks good to me, only a small comments need your help again.

File: site/docs/4.6.0/admin/http.md (hunk 1 above)
Comment (sijie) [reply site/docs/4.6.0/admin/http.md]: nice catch. I fixed the issue.

**Comment by eolivelli** [comment]
I am sending the announce of 4.5.1 @sijie

**Pull request closed without merging**